bokehjs/gulp/tasks/scripts.coffee

```diff
@@ -62,6 +62,7 @@ gulp.task "scripts:tsjs", ["scripts:coffee", "scripts:js", "scripts:ts"], () ->
         gutil.log("#{gutil.colors.red(real)}#{rest}")
         return
 
+      # XXX: can't enable "6133", because CS generates faulty code for closures
```

Comment:
We use coffee `1.10` it looks like the latest release is `1.12.5` is it also broken in newer releases?